filebeat/prospector/docker/config.go

```diff
@@ -0,0 +1,11 @@
+package docker
+
+var defaultConfig = config{
+	ContainersIDs:  []string{},
+	ContainersPath: "/var/lib/docker/containers",
+}
+
+type config struct {
+	ContainersIDs  []string `config:"containers_ids"`
```

Comment:
Could we make this `container.ids` and `container.path` or `containers.ids` and `containers.path`?